.gitignore

```diff
@@ -0,0 +1 @@
+node_modules/
```

Comment:
This red icon means that you have an empty line at the end of the code. You can delete it ;)